### PR TITLE
scVelo 0.1.25

### PR DESCRIPTION
**Pull request by orbsmiv**
For INC1023090

`scVelo-0.1.25-foss-2019b-Python-3.7.4.eb`
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-power9
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/s/scVelo/scVelo-0.1.25-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        'preinstallopts': "export CPLUS_INCLUDE_PATH=$EBROOTFREETYPE/include/freetype2:${CPLUS_INCLUDE_PATH} && ",
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
+        'checksums': ['ffe2f9cdcea1086fc414e82f42271ecf1976700b8edd16ca9d376189c6d93aee'],
+    }),
+    ('numpy-groupies', '0.9.10', {
+        'modulename': 'numpy_groupies',
+        'source_tmpl': 'numpy_groupies-%(version)s.tar.gz',
+        'checksums': ['a6835c252b7bd489e0f701f8bf5bc880c647ef251e3b434998b95a8aecc9b68a'],
+    }),
+    ('seaborn', '0.10.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/s/scVelo/scVelo-0.1.25-foss-2019b-Python-3.7.4.eb:56]: https://bear-apps.bham.ac.uk/applications/Seaborn/0.10.0-foss-2019b-Python-3.7.4/

Comment (orbsmiv) [reply easybuild/easyconfigs/s/scVelo/scVelo-0.1.25-foss-2019b-Python-3.7.4.eb]: Resolved in commit d2f0c3e

**Comment by edmondac** [comment]
numba fails on POWER9:

```
failing sanity check for 'numba' extension: command "python -c "import numba"" failed; output:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/__init__.py", line 30, in <module>
    from .decorators import (cfunc, generated_jit, jit, njit, stencil, jit_module)
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/decorators.py", line 14, in <module>
    from .targets import registry
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/registry.py", line 5, in <module>
    from . import cpu
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/cpu.py", line 11, in <module>
    from .base import BaseContext, PYOBJECT
  File "/rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/targets/base.py", line 18, in <module>
    from numba import _dynfunc, _helperlib
ImportError: /rds/bear-apps/2019b/EL7-power9/software/numba/0.47.0-foss-2019b-Python-3.7.4/lib/python3.7/site-packages/numba/_helperlib.cpython-37m-powerpc64le-linux-gnu.so: undefined symbol: pthread_atfork,  (at easybuild/2019b/src/easybuild-framework/easybuild/framewor
k/easyblock.py:2593 in _sanity_check_step)
```

**Comment by orbsmiv** [comment]
@edmondac I think we had a similar issue with https://github.com/bear-rsg/easybuild-easyconfigs/pull/46, which also never got built on POWER9. (https://bear-apps.bham.ac.uk/applications/numba/0.47.0-foss-2019b-Python-3.7.4/)

**Pull request merged**